R/qenv-eval_code.R

```diff
@@ -28,18 +28,20 @@
 setGeneric("eval_code", function(object, code) standardGeneric("eval_code"))
 
 setMethod("eval_code", signature = c("qenv", "character"), function(object, code) {
-  id <- sample.int(.Machine$integer.max, size = 1)
+  parsed_code <- parse(text = code, keep.source = TRUE)
+  comments <- extract_comments(parsed_code)
+  id <- sample.int(.Machine$integer.max, size = length(parsed_code))
```

Comment:
now all `@id`, `@code`, `@warnings`, `@messages` will have the same length as number of single calls in input `code`. The resulted object will have longer fields (especially code field). We need to double check if that doesn't brake `teal` and `teal.data` and Show R Code in modules.